### PR TITLE
Website versioning

### PR DESCRIPTION
**Pull request by richardsliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/136)

### CODE REVIEW

**Comment by richardsliu** [comment]
This PR adds versioning to the Kubeflow public website. 

I've created a separate branch for kubeflow/website at https://github.com/kubeflow/website/tree/v0.2-branch. This branch is set up with Netlify. 

To preview the changes, see: https://deploy-preview-136--competent-brattain-de2d6d.netlify.com/

In the top right corner, you should see a button for "latest". Click on the button to redirect the page to either latest or v2.0. Currently the two sites should be identical, but going forward any changes made to either master or v2.0-branch should be reflected only on the respective site.

Future stable releases can follow the same process:
1. Add a new branch under kubeflow/website and enable Netlify;
2. Add a new version mapping in config.toml;
3. Configure DNS to point www.kubeflow.org to the latest stable release.

**Comment by jlewi** [comment]
This is great thank.

Can you add relevant details to the README
https://github.com/kubeflow/website/blob/master/README.md

Can you also send another PR to add relevant steps to our release process
https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md

**Comment by richardsliu** [comment]
Fixed sass files and added a section to README.

File: README.md [hunk 1]
```diff
@@ -38,3 +38,22 @@ gulp
...
+* `www.kubeflow.org` always points to the latest *stable* release
+* `latest.kubeflow.org` always points to Github head
+* `vXXX-YYY.kubeflow.org` points to the release at vXXX.YYY-branch
+
+Furthermore, whenever any documents reference any source code, the links should be created
+using the version shortcode, like so:
+```
+https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/scripts/gke/deploy.sh
+```
+This ensures that all the links in a versioned webpage point to the correct branch.
```
Comment (jlewi) [inline README.md]: This is awesome!

**Comment by jlewi** [comment]
/lgtm
/approve

/hold 

@abhi-g Any  additional comments?

Is "latest" too ambiguous? It could be interpreted as "latest stable release". Would it be better to call it "master", "head", "dev", "unstable", "unreleased"

master has the advantage of following the convention of being the branch name.

I think I prefer something that is unambiguous. So I lean towards "unstable"

**Comment by abhi-g** [comment]
Looks good to me.

Let’s go with master. If unstable.kubeflow.org gets picked up, may not look
good.

Abhishek

**Comment by richardsliu** [comment]
Renamed to master.

**Pull request merged**